astropy/io/ascii/tests/test_rst.py

```diff
@@ -29,9 +28,9 @@ def test_read_normal():
     reader = ascii.get_reader(reader_cls=ascii.RST)
     dat = reader.read(table)
     assert_equal(dat.colnames, ["Col1", "Col2"])
```

Comment:
```suggestion
    assert dat.colnames == ["Col1", "Col2"]
```
Here as well, all the "names" comparisons are better done this way